store/postgres/discussion_model.go

```diff
@@ -0,0 +1,57 @@
+package postgres
+
+import (
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/odpf/columbus/discussion"
+)
+
+type DiscussionModel struct {
+	ID        string         `db:"id"`
+	Title     string         `db:"title"`
+	Body      string         `db:"body"`
+	Type      int            `db:"type"`
+	State     int            `db:"state"`
```

Comment:
I do know storing this as number will be much cheaper format in DB but aside from that is there any other concerns?
Storing these values as string may take more space but should be simpler to implement all around the code and hopefully better development/debugging experience.

From my exp, dbsize rarely matters in this case unless im missing something.

Reply:
It is actually not about the db size but more into having no semantic stored in the DB.

It is actually possible to use it as string, let me check.

Reply:
Hmm ok, personally I would go for storing enum as string first until it is not feasible anymore (which rarely happens) because it is easier and quicker to implement.

But since we already have all the codes written then it would be counter productive to revert it again.
Also CMIIW, we need to make sure the order of the enums are not changed or else we could get corrupted data which is hard to fix. (someone's creating a discussion with application having wrong enum order).

Reply:
Sure I am updating it to string, it is not that complex though.

Reply:
Oh it is fine, since you already implemented those. We can keep storing enum as int. @mabdh

Reply:
@StewartJingga already updated it to string 😄

Reply:
oh okay 😅 , sorry for the trouble 🙏 .